patches/chrome-browser-resources-settings-site_settings-constants.js.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/resources/settings/site_settings/constants.js b/chrome/browser/resources/settings/site_settings/constants.js
-index ed0c9d30db323299a0b943b6f8c73c4c171d09f0..1e343f0f14376b3edc45a4c29909c504cbc13620 100644
+index ed0c9d30db323299a0b943b6f8c73c4c171d09f0..ef97879026546b111877258652efc0abdba09565 100644
 --- a/chrome/browser/resources/settings/site_settings/constants.js
 +++ b/chrome/browser/resources/settings/site_settings/constants.js
 @@ -43,6 +43,7 @@ export const ContentSettingsTypes = {
    VR: 'vr',
    WINDOW_PLACEMENT: 'window-placement',
    ZOOM_LEVELS: 'zoom-levels',
-+  AUTOPLAY: 'autoplay',
++  AUTOPLAY: 'autoplay', ETHEREUM: 'ethereum',
```

Comment:
same as above cc @petemill

Reply:
see https://github.com/brave/brave-core/pull/9443#discussion_r672833554

Reply:
I've removed this change, however, I left autoplay untouched cuz I think it should be separated in its own PR.

Reply:
https://github.com/brave/brave-core/pull/9443/commits/4f3311e5755f2853449333620885278f0246423d